### ISSUE TITLE
Bug: File header comments get parsed as comment on first string (JS file format)

### ISSUE THREAD

**Issue by patcon** [box/mojito#588]
Hi! Thanks for this project!

Very low priority "bug":

When processing JS files of proper format, a header comment at the top of the file (like the sort used for copyright) gets included as the comment on the first string. I observed this when there was otherwise no comment on the first string.

## Potential Solution
Don't start parsing strings from the file until the line with "export" on it, or some other representative token where the hash begins.

(I'm not blocked on this, as I believe our project can remove the header comment, so no pressure from me!)
